src/media/js/addon/components/addon.js

```diff
@@ -71,6 +72,13 @@ export class Addon extends React.Component {
           </dd>
         </dl>
 
+        {this.props.description &&
+          <di>
```

Comment:
I brought down the <dl> to include this, but I appear to have lost that in a rebase. I've fixed it in an upcoming commit.